### PR TITLE
Feature: Add pinned, owner and reply from owner indicators for comments

### LINKED ISSUE #1431: Distinguish owner of the video in the comment section
**Is your feature request related to a usage problem (not a bug)? Please describe.**
Sometimes it is difficult for me to see in a fast way where a comment is of the uploader of the video. It would be nice to have some kind of distinct appearance.

**Describe the solution you'd like to see implemented**
See screenshot.

**Describe alternatives you've considered**
None.

**Screenshots**
gray comment profile owner

**Additional context**
None.

### LINKED ISSUE #1432: View replies form the uploader of the video
**Is your feature request related to a usage problem (not a bug)? Please describe.**
Often in informative videos i tend to look for comments of the uploader of the video.
Sometimes he/she clarifies things in a discussion in the comment section.

**Describe the solution you'd like to see implemented**
Change "View x replies" to "View x replies from y and others" ,If the creator of the vid has commented under another comment

**Describe alternatives you've considered**
None.

**Screenshots**
creator replies comment

**Additional context**
None.

### LINKED ISSUE #1433: Add Pinned comments indicator
**Is your feature request related to a usage problem (not a bug)? Please describe.**
As far as i know pinned comments are always displayed above all other comments, but there is no indicator for it.

**Describe the solution you'd like to see implemented**
Add a indicator to know that a comment is pinned.

**Describe alternatives you've considered**
None.

**Screenshots**
pinned

**Additional context**
None.

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
---
Add pinned, owner and reply from owner indicators for comments
---

**Important note**
We may remove your pull request if you do not use this provided PR template correctly.

**Pull Request Type**
Please select what type of pull request this is:
- [ ] Bugfix
- [x] Feature Implementation

**Related issue**
Please link the issue your pull request is referring to. If this pull request fully resolves the relevant issue, put "closes" before the issue number. Example: "closes #123456".
closes #1433 
closes #1432 
closes #1431 

**Description**
This PR adds visual indicators for pinned comments, comments by video owner and comments that were replied to by owner.

**Screenshots (if appropriate)**
Light theme:
image

image

Dark theme:
image

image

Black theme:
image

image

Dracula theme:
image

image

**Testing (for code that is not small enough to be easily understandable)**
Has this pull request been tested?
Please describe shortly how you tested it and whether there are any ramifications remaining. 

**Desktop (please complete the following information):**
 - OS: Windows
 - OS Version: 10
 - FreeTube version: 0.15.0

**Additional context**
Only owner indicator is supported by Invidious

### CODE REVIEW

**Comment by ChunkyProgrammer** [comment]
@jasonhenriquez I'm not sure the best way to make the owner indicator accessible (it only changes the background color if the comment is by an owner). Any ideas?

**Comment by ChunkyProgrammer** [comment]
> oh i overlooked something. This needs to be changed. In the comment section of YT the author is also highlighted but in FT not. Screenshot from 2021-10-11 18-07-13 Screenshot from 2021-10-11 18-07-39
> 
> Edit: Change 'Hide 3 replies from Techlore and others' to Hide 3 replies

Done

image

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
U overlooked what i mentioned in my edit😢

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — DISMISSED** [review]
Lgtm!

**Review by peepopoggers — DISMISSED** [review]
lg

File: src/renderer/components/watch-video-comments/watch-video-comments.vue [hunk 1]
```diff
@@ -97,6 +109,8 @@
             {{ comment.numReplies }}
             <span v-if="comment.numReplies === 1">{{ $t("Comments.Reply").toLowerCase() }}</span>
             <span v-else>{{ $t("Comments.Replies").toLowerCase() }}</span>
+            <span v-if="comment.hasOwnerReplied && !comment.showReplies"> {{ $t("Comments.From $").toLowerCase().replace("$", channelName) }}</span>
```
Comment (PikachuEXE) [inline src/renderer/components/watch-video-comments/watch-video-comments.vue:112]: It feels like this is like "passing variable to translation" (using `replace` is fine)
https://guides.rubyonrails.org/i18n.html#passing-variables-to-translations

So I suggest having named placeholders
```suggestion
            <span v-if="comment.hasOwnerReplied && !comment.showReplies"> {{ $t("Comments.From $channelName").toLowerCase().replace("$channelName", channelName) }}</span>
```

File: static/locales/en-US.yaml [hunk 2]
```diff
@@ -602,16 +603,19 @@ Comments:
...
   Top comments: Top comments
   Newest first: Newest First
-  # Context: View 10 Replies, View 1 Reply
+  # Context: View 10 Replies, View 1 Reply, View 1 Reply from Owner, View 2 Replies from Owner and others
   View: View
   Hide: Hide
   Replies: Replies
   Show More Replies: Show More Replies
   Reply: Reply
+  From $: From $
```
Comment (PikachuEXE) [inline static/locales/en-US.yaml:612]:
```suggestion
  From $channelName: From $channelName
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]

File: src/renderer/components/watch-video-comments/watch-video-comments.vue (hunk 1 above)
Comment (ChunkyProgrammer) [reply src/renderer/components/watch-video-comments/watch-video-comments.vue:112]: That makes a lot of sense to me but it also only uses the "$" in locales for other key values so I think a PR should probably be made to update it sometime in the future

**Review by peepopoggers — DISMISSED** [review]
LGTM after testing locally

File: src/renderer/components/watch-video-comments/watch-video-comments.vue (hunk 1 above)
Comment (PikachuEXE) [reply src/renderer/components/watch-video-comments/watch-video-comments.vue]: Sure, I will make that PR

**Review by PikachuEXE — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
U SHALL NOT BE MERGE
Capture
D

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
Oops I did not test after change

**Comment by peepopoggers** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc :'((

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]
LOOKKSSS VERYYY GOOD MY FRIEND! :)

**Review by peepopoggers — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE use ur Approve Request ability to inflict a super effective merge!

**Review by PikachuEXE — APPROVED** [review]
Pikapi~

**Pull request merged**